data_steward/validation/main.py

```diff
@@ -364,6 +390,14 @@ def generate_metrics(hpo_id, bucket, folder_prefix, summary):
         report_data[report_consts.COMPLETENESS_REPORT_KEY] = query_rows(
             completeness_query)
 
+        # participant validation metrics
+        logging.info(f"Ensuring participant validation can be run for {hpo_id}")
+        duplicate_tables = check_duplicates_and_validate(hpo_id, report_data)
+        if not duplicate_tables:
```

Comment:
@nishanthpp93 so the participant validation runs only when there are no duplicates. Can we add messaging about the issue in results.html and is there a decision on how we are going to approach this?

Reply:
Yeah, that makes sense. I didn't want to add the new section in results.html until we know what metrics should go there. I think EHR Ops will come up with metrics to figure out what needs to be shown to sites. Right now, there will be a logging message printed if it was skipped.

Reply:
I don't think the sites anticipate this yet. @jp3477 Should we start showing warnings in the results.html related to participant validation or hold off on it?

Reply:
@nishanthpp93 , we should hold off. We're still deciding how we would want to display it.